pkg/doc/signature/proof/data.go

```diff
@@ -19,6 +22,32 @@ type signatureSuite interface {
 	GetDigest(doc []byte) []byte
 }
 
+// SignatureRepresentation defines a representation of signature value.
+type SignatureRepresentation int
+
+const (
+	// SignatureProofValue uses "proofValue" field in a Proof to put/read a digital signature.
+	SignatureProofValue SignatureRepresentation = iota
+
+	// SignatureJWS uses "jws" field in a Proof as an element for representation of detached JSON Web Signatures.
+	SignatureJWS
+)
+
+// CreateVerifyData creates data that is used to generate or verify a digital signature.
+// It depends on the signature value holder type.
+// In case of "proofValue", the standard Create Verify Hash algorithm is used.
+// In case of "jws",
+func CreateVerifyData(suite signatureSuite, jsonldDoc map[string]interface{}, proof *Proof) ([]byte, error) {
```

Comment:
CreateVerifyData and CreateVerifyHash have same meaning to create data for signature (creating this data involves hashing thus CreateVerifyHash). We should probably have only one function,  let's discuss.

Reply:
Yes, agree, it makes sense. The reason I added a separate method is that CreateVerifyHash is different in some steps in the case of JWS. I described it above https://github.com/hyperledger/aries-framework-go/pull/1137#discussion_r374120358.